<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>com::sptci::Watermark</class>
 <widget class="QWidget" name="com::sptci::Watermark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watermark Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/images/Icon.ico</normaloff>:/images/Icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="imageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="text">
         <property name="placeholderText">
          <string>Sample watermark</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="fontSize">
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Font Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Font Face</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="fontFilters">
          <set>QFontComboBox::MonospacedFonts|QFontComboBox::ProportionalFonts|QFontComboBox::ScalableFonts</set>
         </property>
         <property name="currentFont">
          <font>
           <family>Abadi MT Condensed Extra Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Font Colour</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCommandLinkButton" name="fontColour">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Pick colour</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Text Opacity</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="textOpacity">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="position">
         <item>
          <property name="text">
           <string>Top</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Middle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Alignment</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="alignment">
         <item>
          <property name="text">
           <string>Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Centre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Background Colour</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCommandLinkButton" name="backgroundColour">
         <property name="text">
          <string>Pick colour</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Background opacity</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="backgroundOpacity">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>277</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="preview">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAs">
         <property name="text">
          <string>Save As...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyAll">
         <property name="text">
          <string>All files...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="allInDirectory">
         <property name="text">
          <string>In directory...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fontColour</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>selectFontColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>845</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preview</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundColour</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>selectBackgroundColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAs</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allInDirectory</sender>
   <signal>clicked()</signal>
   <receiver>com::sptci::Watermark</receiver>
   <slot>allInDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
